test/unit/check/property/Property.spec.ts

```diff
@@ -3,11 +3,14 @@ import { Shrinkable } from '../../../../src/check/arbitrary/definition/Shrinkabl
 import { property } from '../../../../src/check/property/Property';
 import { pre } from '../../../../src/check/precondition/Pre';
 import { PreconditionFailure } from '../../../../src/check/precondition/PreconditionFailure';
+import { configureGlobal, resetConfigureGlobal } from '../../../../src/check/runner/configuration/GlobalParameters';
 
 import * as stubArb from '../../stubs/arbitraries';
 import * as stubRng from '../../stubs/generators';
 
 describe('Property', () => {
+  beforeEach(() => resetConfigureGlobal());
```

Comment:
Same use `afterEach` instead

Reply:
```suggestion
  afterEach(() => resetConfigureGlobal());
```

I think it would be safer to always clean properly after the run. Cleaning before do not ensure we clean stuff for next tests.